WordPressShared.podspec

```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
   s.homepage      = "https://github.com/wordpress-mobile/WordPress-iOS-Shared"
   s.license       = "GPLv2"
   s.author        = { "WordPress" => "mobile@automattic.com" }
-  s.platform      = :ios, "11.0"
+  s.platform      = :ios, "13.0"
```

Comment:
:nod:

You said in the PR description that you updated the requirement to iOS 12 but here you're setting it to iOS 13.

I think that's the right move, because `NSRelativeDateFormatter` is iOS13+ so would not work in iOS12. But just wanted to highlight that, given your PR description might set the wrong expectations.

Reply:
Ah yep, good call. I went back and upgraded to iOS 13 last week since now all our projects are there now (they weren't when this PR was first opened) and forgot to update the PR message. Thanks for flagging!